WeatherApp/WeatherApp/Feature/CityListScreen/CityListViewModel.swift

```diff
@@ -2,25 +2,29 @@ import SwiftUI
 
 class CityListViewModel: ObservableObject {
 
-    @Published var cities: [City] = [
-        City(name: "Zagreb"),
-        City(name: "Paris"),
-        City(name: "New York"),
-        City(name: "Tokyo"),
-        City(name: "London"),
-        City(name: "Los Angeles"),
-        City(name: "Toronto"),
-        City(name: "Split")]
-
-    let appearance = UINavigationBarAppearance()
+    @Published var cities: [City] = [] {
+        didSet {
+            storeCitiesUseCase.storeCities(cities: cities)
+        }
```

Comment:
no a fun of didset as it can mask what is really happening, better call what is needed when you set the cities property